lib/file/monspeak.txt

```diff
@@ -909,10 +909,10 @@ says, 'Located a target, I'll begin to eliminate...!'
 says, '"The Ship That Would Not Die" won't go down that easy...'
 
 N:1362:Mika Misono, a witch of the Tea Party
-says, 'Hi, can you get out of my way? ...Can't? Okay, I'll have you get out forcefully.'
-says, 'Saori is... there probably, isn't?'
-says, 'I guess you will kill me, will not? Hmm, that's too bad...'
-is seeking someone by her eyes that a thirst for deprive all from it...
+says, 'Hi, can you get out of my way? ...Can't? Okay, I'll have to force you then.'
+says, 'Saori is... there probably, isn't she?'
+says, 'I guess you will kill me, will you? Hmm, that's too bad...'
+is looking for the person whose eyes seem to have stolen everything from her...
```

Comment:
I tried something that was between the original translation and what Google translate gave for the Japanese text ("is searching for someone with eyes that  seem to have stolen everything from him...").  It sounds like you want something like "is focused on finding the person who took everything from her...".

Reply:
I understood.
Then I'd translate "is looking for a certain one for to take everything from the one..." Is that right?

Reply:
Okay, though to my ears "person" rather than "one" sounds better, "that" rather than "the" is clearer, and simplify "for to" to "to".  So, I'd use "is looking for a certain person  to take everything from that person...".

Reply:
Sounds good.
I have no further comment.